styles/styles.css

```diff
@@ -0,0 +1,106 @@
+@import url('https://elixir-ui.netlify.app/Components/elixir.css');
+@import url('./variables.css');
+@import url('./reset.css');
+
+.header {
+    height: 10vh;
+}
+
+.main {
+    justify-content: center;
+    min-height: calc(100vh - 10vh - var(--space-3));
+}
+
+.quiz-categories {
+    display: grid;
+    grid-template-columns: repeat(auto-fit, minmax(200px, 1fr));
+    gap: var(--space-2);
+    place-items: center center;
+    width: 100%;
+}
+
+.card {
+    width: 100%;
+    height: 100%;
+    max-height: 400px;
+    position: relative;
+    background-color: transparent;
+    border-width: 2px;
+    transition: transform 0.3s ease;
+    -webkit-transition: transform 0.3s ease;
+    -moz-transition: transform 0.3s ease;
+    -ms-transition: transform 0.3s ease;
+    -o-transition: transform 0.3s ease;
+}
+
+.card:hover {
+    transform: scale(1.025);
+    -webkit-transform: scale(1.025);
+    -moz-transform: scale(1.025);
+    -ms-transform: scale(1.025);
+    -o-transform: scale(1.025);
+}
+
+.card-header {
+    height: 100%;
+}
+
+.card-img {
+    border-radius: var(--br-4);
+    height: 100%;
+    object-fit: cover;
+    object-position: top center;
+    -webkit-border-radius: var(--br-4);
+    -moz-border-radius: var(--br-4);
+    -ms-border-radius: var(--br-4);
+    -o-border-radius: var(--br-4);
+}
+
+.card-absolute {
+    transition: all 0.3s ease-in-out;
+    justify-content: center;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    background-color: rgba(0,0,0,0.7);
+    position: absolute;
+    top: 0;
+    bottom: 0;
+    left: 0;
+    right: 0;
```

Comment:
You can probably add inset property here as shorthand for top, bottom, left, and right properties

Reply:
Thanks for this suggestion. I have made the changes!